src/docs/product/issues/issue-details/performance-issues/regressions.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Function Regression"
+sidebar_order: 40
+description: "Learn more about Function Regression issues and how to diagnose and fix them."
+---
+
+Function Regression issues capture a generic class of problems where the duration of a function has increased over time. This can lead to degraded performance in your application. Sentry attempts to monitor some of your most common functions out of the box and report any possible regressions under the Function Regression issue type.
+
+## Detection Criteria
+
+[Profiling](/product/profiling) must be enabled for Sentry to detect Function Regression issues. Once set up, Sentry will periodically check the 95th percentile function duration of the most common application functions in your project. When a significant change in the p95 value for function duration is detected and has been sustained for some time, a regression issue is reported.
+
+## Charts
```

Comment:
Missing screenshots?

Reply:
Will be added for GA as we're still polishing the UI